packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/CredentialsTests.kt

```diff
@@ -60,7 +60,7 @@ class CredentialsTests {
 
     @BeforeTest
     fun setup() {
-        app = TestApp()
+        app = TestApp(this::class.simpleName)
```

Comment:
Isn't there a way to get the actual name of the test instead of the suite name?

Reply:
I created https://github.com/realm/realm-kotlin/issues/1496 to track this